cron.yaml

```diff
@@ -0,0 +1,5 @@
+cron:
```

Comment:
Since we already have a cron.yaml.example we should either
- get the contents from that example file into this cron.yaml and remove the example. This would also require to update the *make deploy* target Makefile because right now using cron is optional.
or
- move this into the example, so that people can decide whether to remove old links or keep them

I would vote for moving this into the example file + include a comment what this is for.